packages/core/addon/templates/components/number-format-dropdown.hbs

```diff
@@ -6,6 +6,6 @@ terms. --}}
   {{/dd.trigger}}
 
   {{#dd.content class="number-format-dropdown__container"}}
-    {{number-format-selector format=format onUpdateFormat=(action (mut format))}}
+    {{number-format-selector format=format onUpdateFormat=(action (set this "format" _))}}
```

Comment:
Can we make these
```suggestion
    {{number-format-selector format=format onUpdateFormat=(action (set this.format _))}}
```

Reply:
I also prefer that syntax, but it seems it doesn't work right now.
https://github.com/pzuraq/ember-set-helper/issues/8

Reply:
Oh :(

Reply:
The first example in the docs using that sort of 2 argument syntax, but the function definition seems to explicitly require 3 arguments. This will have to do for now until that issue is fixed.